UDV-Core/src/Modules/Temporal/viz.js

```diff
@@ -0,0 +1,86 @@
+import * as vis from 'vis-network';
+
+/**
+* Manager for the graph
+* Done as a singleton so only one instance can exist.
+* If a constructor is called :
+*   - First time = instantiation
+*   - Other time = return the already existing NetworkManagerSingleton
+*/
+export class NetworkManagerSingleton {
```

Comment:
Why is it named `NetworkManagerSingleton` ? (in particular, why 'network'?)

Reply:
It is the manager of the network (a name for this specific king of graphe)

Reply:
Can you add this in the doc then please ? And explain why you had to create a new object on top of vis-network ? (i.e. what was missing in this lib that you had to create it)

Reply:
Added